backend/src/routes/ansibletower.ts

```diff
@@ -13,6 +13,9 @@ interface AnsibleCredential {
   token: string
 }
 
+// must match ansiblePaths in frontend/src/resources/utils/resource-request.ts
+const ansiblePaths = ['/api/v2/job_templates/', '/api/v2/workflow_job_templates/']
+
 export function ansibleTower(req: Http2ServerRequest, res: Http2ServerResponse): void {
   const token = getToken(req)
   if (!token) return unauthorized(req, res)
```

Comment:
Are we going to check if the token is valid? That was the complaint of the defect.

Reply:
And this was what I was getting at in terms of checking all our APIs to make sure we handle them consistently.